app/assets/stylesheets/admin/_form.scss

```diff
@@ -0,0 +1,9 @@
+.form {
+  .event_description textarea {
+    height: 6rem;
+  }
+
+  .event_menu textarea {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `event_menu` should be written in lowercase with hyphens